src/sentry/tasks/code_owners.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from sentry import features
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger("sentry.tasks.code_owners")
+
+
+@instrumented_task(
+    name="sentry.tasks.update_code_owners_schema",
+    queue="code_owners",
+    default_retry_delay=5,
+    max_retries=5,
+)
+def update_code_owners_schema(organization=None, integration=None, projects=None, **kwargs):
+    from sentry.models import ProjectCodeOwners, RepositoryProjectPathConfig
+
+    if not features.has("organizations:integrations-codeowners", organization):
+        return
+    try:
+        code_owners = []
+
+        if projects:
+            code_owners = ProjectCodeOwners.objects.filter(project__in=projects)
+
+        if organization and integration:
```

Comment:
then here could just be checking for the integration. basically it looks like either you have the `projects` or you have the `integration`